### ISSUE TITLE
Problem when installing with apt

### ISSUE THREAD

**Issue by jylenhofdecat** [jdx/rtx#974]
**Describe the bug**

Seems something is wrong when pushing the version to your debian repo....  I cannot install it

Err:8 https://rtx.pub/deb stable/main amd64 Packages
  File has unexpected size (358 != 359). Mirror sync in progress? [IP: 104.26.3.174 443]
  Hashes of expected file:
   - Filesize:359 [weak]
   - SHA256:8fb487ae18ff82e94a9753f18ce6359f3baeedebcf7015e596446b12145dea95
   - SHA1:3b081bae1f2e6682780947d43283bda624ab44b1 [weak]
   - MD5Sum:20325921aaf9ed31284aa1d57adf2d64 [weak]
  Release file created at: Sun, 15 Oct 2023 23:20:17 +0000
Fetched 2062 B in 1s (2817 B/s)
Reading package lists... Done
E: Failed to fetch https://rtx.pub/deb/dists/stable/main/binary-amd64/Packages.gz  File has unexpected size (358 != 359). Mirror sync in progress? [IP: 104.26.3.174 443]
   Hashes of expected file:
    - Filesize:359 [weak]
    - SHA256:8fb487ae18ff82e94a9753f18ce6359f3baeedebcf7015e596446b12145dea95
    - SHA1:3b081bae1f2e6682780947d43283bda624ab44b1 [weak]
    - MD5Sum:20325921aaf9ed31284aa1d57adf2d64 [weak]
   Release file created at: Sun, 15 Oct 2023 23:20:17 +0000
E: Some index files failed to download. They have been ignored, or old ones used instead.

**Comment by jdx** [comment]
Hmm I made some caching changes on the server which may be causing this. Is it still happening?

**Comment by jylenhofdecat** [comment]
It's OK now
Closing

**Comment by cjntaylor** [comment]
This is unfortunately still happening (or happening again?). Any chance there's some sort of CDN/cache propagation issue?

```
#18 0.850 Get:8 https://rtx.pub/deb stable/main amd64 Packages [357 B]
#18 0.851 Err:8 https://rtx.pub/deb stable/main amd64 Packages
#18 0.851   File has unexpected size (360 != 357). Mirror sync in progress? [IP: 104.26.3.174 443]
#18 0.851   Hashes of expected file:
#18 0.851    - Filesize:357 [weak]
#18 0.851    - SHA256:af77f17d2642df66717897010bf8e105b8fcce3a59ed3dffb7788b85fe2be019
#18 0.851    - SHA1:2b8e744356f7828055c85190aec5a49dbdca2199 [weak]
#18 0.851    - MD5Sum:88a81f2c049bd162c3acf11ebb508ac9 [weak]
#18 0.851   Release file created at: Thu, 09 Nov 2023 18:47:27 +0000
```

**Comment by jdx** [comment]
I found out the cache purge was actually failing but I didn't have curl emit an error. This should now be resolved now.

**Comment by jdx** [comment]
I changed the hostname for that server to rtx.jdx.dev. The old rtx.pub host should continue working for the foreseeable future but it might be good to switch to rtx.jdx.dev since that's what I'll be focusing on generally.

That said, I know what the issue is. I trigger a cache purge when releasing rtx but I switched it over to jdx.rtx.dev from rtx.pub but what I should've done is trigger a cache purge on _both_ since people still using the old one will get the wrong assets.
